Data/future/cme/margins/GE.csv

```diff
@@ -17,26 +17,67 @@ date,initial,maintenance
 20100813,935,850
 20110621,825,750
 20110822,770,700
-20120117,742.5,675
-20120217,687.5,625
+20120117,743,675
+20120217,688,625
+20130102,358,325
 20130115,605,550
+20130116,275,250
 20130305,550,500
+20130306,220,200
 20130621,605,550
+20130624,220,200
 20130702,715,650
+20130703,220,200
 20130710,880,800
+20130711,220,200
 20130716,495,450
+20130717,220,200
 20130910,451,410
+20130911,176,160
 20130917,770,700
+20130918,176,160
 20140321,638,580
-20140613,797.5,725
+20140324,176,160
+20140613,798,725
+20140616,176,160
 20140919,715,650
-20141017,632.5,575
+20140922,176,160
```

Comment:
Pretty sporadic, though it seems standard for this contract. If I remember correctly, the Eurodollar contract's margins apply to tiers of contracts. Maybe the changes in tiers causes the value to be very volatile 🤔